strapi/.env.example

```diff
@@ -0,0 +1,6 @@
+HOST=0.0.0.0
+PORT=1337
+APP_KEYS="toBeModified1,toBeModified2"
+API_TOKEN_SALT=tobemodified
+ADMIN_JWT_SECRET=tobemodified
+JWT_SECRET=tobemodified
```

Comment:
This file should not be part of git, to avoid exposing sensitive keys, or use environment variables (like strapi/config/admin.ts)

Reply:
You would be right if this was the actual `.env` file, however note that the latter is already ignored:
https://github.com/clicepfl/clic-website-v2/blob/1291c61c9dd4221b9fc309932c3d6fa877d7e3ca/strapi/.gitignore#L109
This file provides an example so that people can clone the repo and generate their own corresponding keys 😊 the idea is to hide the actual secrets, not the shape of the config